### PR TITLE
hotfix

### PR DESCRIPTION
**Pull request by HassanAmr**
modified exit code for help argument to 0, and modified the Makefile to include an additional include directory, and the fopenmp flag

### CODE REVIEW

**Comment by bgruening** [comment]
The exit() at https://github.com/HassanAmr/INFO-RNA/blob/d0fa08a51647479777559f6f0258822ef3d92893/inv_folding_const.cpp#L94 ?

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ VIENNA          = /home/Vienna_1.6
 ###############################################################
 
 
-CXXFLAGS        = -O4 -Wall -g -I$(VIENNA)/include/ViennaRNA 
+CXXFLAGS        = -O4 -Wall -g -fopenmp -I$(VIENNA)/include/ViennaRNA -I$(VIENNA)/include
```
Comment (bgruening) [inline Makefile:14]: Are you sure about the oponmp support, I don't think infor-rna supports this? ping @martin-mann

Comment (HassanAmr) [reply Makefile:14]: It failed to compile without it when doing the bioconda package.

Comment (bgruening) [reply Makefile:14]: Oh good to know. Lets wait for @martin-mann.
But as the tests are passing I'm fine with merging.

**Comment by HassanAmr** [comment]
The exit() in line 94 was already there. should I remove it?

File: Makefile (hunk 1 above)
Comment (martin-raden) [reply Makefile:14]: INFORNA was coded 10 years back, when the Vienna package had no openmp support.
So basically it is only tested up to Vienna v1.8.5 (for which the CXXFLAGS are fine).

Running it with recent Vienna versions will either require an according configure call for the Vienna package compilation (to switch off openmp) or we could add the suggested fix.

Theoretically, one can also add an according Vienna version check into the configure.ac, but no sure if really needed.

So all in all: fine for me. Should not cause problems as far as I can see.

**Comment by bgruening** [comment]
@HassanAmr line 94 contains a `exit(1);` and the funciton is called `usgae()` so I don't think this should return an error level of 1.
@martin-mann ?

**Comment by martin-raden** [comment]
Concerning "exit(1)" in L94: unfortunately this function is called in error case all over the code and expects an exit != 0.

So best leave it, even if no good coding style. Was one of the first projects of the original INFO-RNA author who is no programmer.

**Comment by bgruening** [comment]
Ok fair enough!

**Comment by HassanAmr** [comment]
Thanks, @martin-mann and @bgruening :)

**Pull request merged**